src/pudl/package_data/notebooks/how_to_access_pudl.ipynb

```diff
@@ -0,0 +1,222 @@
+{
```

Comment:
<p>My guess is that anyone who isn't into databases, which will probably include a lot of people that want to use the output objects, won't immediately understand what "denormalized tables" means, and we should probably offer a definition, or use more descriptive language.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/844/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>